sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -791,11 +791,22 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
     }
   }
 
+  // These table properties should not be included in the output statement of SHOW CREATE TABLE
+  val excludedTableProperties = Set(
+    // The following are hive-generated statistics fields
+    "COLUMN_STATS_ACCURATE",
+    "numFiles",
+    "numPartitions",
+    "numRows",
+    "rawDataSize",
+    "totalSize"
+  )
```

Comment:
This PR is for fixing a bug. We might need to backport to 2.0. When we implementing the translation layer, we can do that, just like what we did for the property names of the Data Source Table schema